test/WebSites/ViewComponentWebSite/Startup.cs

```diff
@@ -15,7 +16,13 @@ public void Configure(IApplicationBuilder app)
             {
                 services.AddMvc(configuration);
             });
-            app.UseMvc();
+
+            app.UseMvc(routes =>
+            {
+                routes.MapRoute(
+                    name: "default",
+                    template: "{controller=Home}/{action=Index}/{id?}");
```

Comment:
hmm, doing this enough that I'm wondering which of the following is `true`
- we tend to write demo-ware rather than realistic test sites
- in either MVC or Routing, we should provide a `public static IRouteBuilder MapDefaultMvcRoute(this IRouteBuilder routeCollectionBuilder)` extension?